docs/compendium/bindings.md

```diff
@@ -0,0 +1,76 @@
+# Bindings
+
+Bindings make static scientific papers interactive by providing better access to the underlying source code and dataset.
+Bindings connect the paper, the dataset, and the source code used to compute a specific figure, numerical result, or table with user interface widgets such as slider to manipulate a value shown in the figure. 
+
+## Purposes
+
+The sort of interactivity depends on the purpose. Here, we support the following purposes:
+
+1. Show the dataset underlying the paper.
+2. Show the source code underlying the paper.
+3. Show the dataset underlying a specific numerical result.
+4. Show the dataset underlying a figure.
+5. Show the source code underlying a specific numerical result.
+6. Show the source code underlying a figure.
+7. Manipulate a parameter used to compute a specific numerical result.
+8. Manipulate a parameter used to compute a figure.
+9. Select data subsets used to produce a figure.
+10. Select data subsets used to produce a specific numerical result.
+
+## Create a new binding
+
+When creating a new compendium, authors will reach the "Create bindings section".
```

Comment:
This is the API documentation, the "Create bindings section" is part of the reference user interface.

What are the prerequisites for a binding from an API perspective? Does the compendium have to be a candidate, or not?

Reply:
"Create bindings section" is not about explaining how it works but which info is needed - reworded

Included prerequisites